hd/lang/lex_utf8.txt

```diff
@@ -1,3 +1,260 @@
+    warning refresh
```

Comment:
We should be able to use digest of previous image in order to avoid this warning message and to reject double submission. See https://caml.inria.fr/pub/docs/manual-ocaml/libref/Digest.html#VALfile

Reply:
Dont think so. This is a "legitimate" redo of a swap command. It is not a "resubmission" of the same image!

Reply:
The real problem here is a cache issue! Portraits have a standard name (fn.oc.sn), which does not change when changing the portrait image. Thus, the pop-up displays the old version unless a refresh screen is performed. But doing a refresh just after the swap will in fact redo the swap!! A better solution to avoid this cache problem is needed!

Reply:
Si le formulaire inclus un digest de l'image `fn.oc.sn` alors re-soumettre le meme formulaire ne re-produira pas le swap car coté serveur nous pouvons vérifier que l'image `fn.oc.sn` actuelle ne correpond pas à celle qui était là lors de la soumission de la création du formulaire.

Donc ça règle un des problèmes. La même chose est utilisé lorsqu'on modifie une personne pour s'assurer que les modifications sont apportées sur une version à jour.